.github/workflows/changelog.yml

```diff
@@ -0,0 +1,29 @@
+name: Changelog
+on:
+  push:
+    branches:
+      - issue-78
```

Comment:
@eddiejaoude Why did you set the branch to `issue-78` specifically?

Reply:
I think that was for testing purposes

Reply:
Yep, I need to update this, thank you 👍